src/main/resources/application-prod.yml

```
@@ -0,0 +1,15 @@
+spring:
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    url: 'jdbc:postgresql://pg-f8s8l.vpc-pub-cdb-kr.ntruss.com:5432/nexters'
+    username: ${DB-USER}
+    password: ${DB-PASSWORD}
+  jpa:
+    hibernate:
+      ddl-auto: update
```

`ddl-auto: update`는 배포 환경에 권장되지 않는 설정인데, 혹시 설정한 이유가 따로 있는지 궁금합니다.
참고: https://stackoverflow.com/questions/221379/hibernate-hbm2ddl-auto-update-in-production

엔티티 설정에 맞게 적용되는 옵션이라 자칫하다가 ORM 컬럼 매핑이 의도치 않게 과하게 작동될 수 있을 것 같아서 `ddl-auto: validate`가 더 나아보이는데 어떻게 생각하시나요?
```suggestion
      ddl-auto: validate
```

---

일단 디비에 테이블 만들어두려고 넣은건데 어느정도 테이블 다 만들어지면 변경하면 될것같습니다.

---

현재는 사실상 prod 서버가 아닌 dev 서버에 적용할 yml이니 `update`로 둬도 괜찮을 것 같네요.

8주차쯤에 validate로 바꾸는 것도 괜찮겠군요

---

저도 릴리즈 전까지는 지금 그대로 가도 괜찮은 것 같아용. create-drop도 되고요.

---

릴리즈할 때 되면 validate + flyway까지 셋업하면 좋을 것 같아요. 얘기해보고 다들 괜찮다 싶으면 제가 셋업할께요